src/main/groovy/life/qbic/datamodel/datasets/datastructure/folders/nanopore/Fast5FailFolder.groovy

```diff
@@ -3,33 +3,33 @@ package life.qbic.datamodel.datasets.datastructure.folders.nanopore
 import life.qbic.datamodel.datasets.datastructure.folders.DataFolder
 
 /**
- * A special case of a DataFolder, its name is always fast5_fail.
+ * A special case of a DataFolder, its name contains always fast5_fail.
```

Comment:
It is `fast5_fail` and does not contain it.

Reply:
I agree with Tobi